src/Modera/BackendSecurityBundle/Controller/UsersController.php

```diff
@@ -144,9 +144,9 @@ public function getConfig()
                 }
 
                 if (isset($params['sendPassword']) && $params['sendPassword'] != '') {
-                    $this->getPasswordManager()->encodeAndSetPasswordAndThenEmailIt($user, $plainPassword);
+                    $this->getPasswordManager()->encodeAndSetPasswordAndThenEmailIt($user, $plainPassword, true);
```

Comment:
By adding a third argument to "encodeAndSetPasswordAndThenEmailIt" and "encodeAndSetPassword" we are enforcing API consumer's requirements onto the universal class (PasswordManager). What probably would be more right in this case is to make PasswordManager throw an exception which contain a property with all errors occurred during validation and here we just could catch this exception and pick a first error. Something like that:

```
try {
    $this->getPasswordManager()->encodeAndSetPassword($user, $plainPassword);
} catch (BadPasswordException $e) {
    throw new BadPasswordException($e->getErrors()[0], null, $e);
}
```